core/tests/protractor_utils/ExplorationEditorSettingsTab.js

```diff
@@ -77,9 +73,8 @@ var ExplorationEditorSettingsTab = function() {
   };
 
   this.enableCorrectnessFeedback = async function() {
-    expect(await enableCorrectnessFeedbackButton.isDisplayed()).toBe(true);
     await action.click(
-      'Enable Correctness Feedback Button', enableCorrectnessFeedbackButton);
+      'Enable Correctness Feedback button', enableCorrectnessFeedbackButton);
   };
 
   this.expectAvailableFirstStatesToBe = async function(names) {
```

Comment:
Below, can we check that the element is visible before calling `.getText()`?

Reply:
Done